<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QDialog" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>730</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>700</height>
   </size>
  </property>
  <property name="font">
   <font>
    <kerning>false</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>711</width>
     <height>631</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(:/icons/images/book.png);
border-radius:10%;

</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>110</y>
       <width>321</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Copperplate</family>
       <pointsize>33</pointsize>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>Auckland Library</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="username">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>260</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:10px;
padding-left:5px;</string>
     </property>
     <property name="cursorPosition">
      <number>0</number>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="usernameLabel">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>260</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Copperplate</family>
       <pointsize>15</pointsize>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="password">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>330</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:10px;
padding-left:5px;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="passwordLabel">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>340</y>
       <width>75</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Copperplate</family>
       <pointsize>15</pointsize>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="memberLogin">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>420</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Copperplate</family>
       <pointsize>-1</pointsize>
       <italic>false</italic>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(58, 64, 90);
color:white;
border-radius:20px;
font: 15px;</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="notmember">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>520</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>.AppleSystemUIFont</family>
       <pointsize>15</pointsize>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>Not a member?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="registration">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>510</y>
       <width>90</width>
       <height>34</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
       <underline>true</underline>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;</string>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="back_3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>420</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Copperplate</family>
       <pointsize>-1</pointsize>
       <italic>false</italic>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;
border-radius:20px;
font: 15px;</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="login_status">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>200</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:red;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(:/icons/images/book.png);
border-radius:10%;

</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>90</y>
       <width>321</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Copperplate</family>
       <pointsize>33</pointsize>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>Auckland Library</string>
     </property>
    </widget>
    <widget class="QLabel" name="usernameLabel_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>200</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Copperplate</family>
       <pointsize>15</pointsize>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLabel" name="passwordLabel_2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>270</y>
       <width>75</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Copperplate</family>
       <pointsize>15</pointsize>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="registerButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>540</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Copperplate</family>
       <pointsize>-1</pointsize>
       <italic>false</italic>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(58, 64, 90);
color:white;
border-radius:20px;
font: 15px;</string>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="emailLabel">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>390</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Copperplate</family>
       <pointsize>15</pointsize>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QLabel" name="phoneLabel">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>450</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Copperplate</family>
       <pointsize>15</pointsize>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>Phone</string>
     </property>
    </widget>
    <widget class="QLabel" name="nameLabel">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>320</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Copperplate</family>
       <pointsize>15</pointsize>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="username_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>200</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:10px;
padding-left:5px;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="password_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>260</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:10px;
padding-left:5px;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="name">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>320</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:10px;
padding-left:5px;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="email">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>380</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:10px;
padding-left:5px;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="phone">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>440</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:10px;
padding-left:5px;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="back_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>540</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Copperplate</family>
       <pointsize>-1</pointsize>
       <italic>false</italic>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;
border-radius:20px;
font: 15px;</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="adminPage">
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(:/icons/images/book.png);
border-radius:10%;

</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>110</y>
       <width>321</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Copperplate</family>
       <pointsize>33</pointsize>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>Auckland Library</string>
     </property>
    </widget>
    <widget class="QLabel" name="adminUse">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>260</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Copperplate</family>
       <pointsize>18</pointsize>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>Admin Use Only</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="admin">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>340</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:10px;
padding-left:5px;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="adminLogin">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>440</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Copperplate</family>
       <pointsize>-1</pointsize>
       <italic>false</italic>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(58, 64, 90);
color:white;
border-radius:20px;
font: 15px;</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="back">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>440</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Copperplate</family>
       <pointsize>-1</pointsize>
       <italic>false</italic>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;
border-radius:20px;
font: 15px;</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="login_status_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>210</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:red;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
